### DISCUSSION TITLE
Wireguard using Mullvad using a single VPN IP address

### DISCUSSION THREAD

**Discussion by beyondunreal79** [qdm12/gluetun#1452] [Q&A] [answered]
Hi, 
I am using Mullvad VPN with a wireguard config. I selected a particular country, city and a single server. I noticed the IP I get seems to changed occasionally, especially if I restart the container. Is there any way to keep it connected to a single IP? Here is my compose.
https://pastebin.com/F328HyBV

**Comment by hmdrmch** [comment]
I had the same issue but with AirVPN as provider.
It changed IP and connected to new server everytime the container was restarted.

I tried to set it up as a custom provider instead of specifying airvpn as provider and that seems to have done the trick.
Now it connect to the same server and gets the same IP everytime the container is restarted.

**Comment by qdm12** [comment, marked as answer]
See https://github.com/qdm12/gluetun/discussions/1492#discussioncomment-5487581 the same applies for all providers. You might want to use `SERVER_HOSTNAMES` that will pin it down to usually one server machine and you should always get the same  public ip address
